docs/modules/user-guide/partials/proc_defining-kubernetes-resources.adoc

```diff
@@ -43,7 +43,7 @@ components:
       uri: ../relative/path/postgres.yaml
 ----
 
-The preceding component references a file that is relative to the location of the devfile. Therefore, this devfile is only loadable by a factory. You supply the location of your devfile to the factory, and then use your devfile to determine the location of the referenced Kubernetes resource list.
+The preceding component references a file that is relative to the location of the devfile. Therefore, you can only load this devfile through a factory. You supply the location of your devfile to the factory, and then use your devfile to determine the location of the referenced Kubernetes resource list.
```

Comment:
through a factory / using a factory
? :) Otherwise, LGTM

Reply:
Thanks, Michal, I'll make this small change once I apply SME feedback :)

Reply:
I removed the references to factory all together. I believe factory is specific only to Che.